lib/features/bottom_nav/view/bottom_nav_view.dart

```diff
@@ -20,11 +20,11 @@ class _BottomNavViewState extends State<BottomNavView> {
     final screenUtil = ScreenUtil();
     return AutoTabsScaffold(
       backgroundColor: const Color(0xFFf2e9e0),
-      routes: const [
-        HomeRoute(),
-        LocationRoute(),
-        OrderRoute(),
-        ProfileRoute()
+      routes: [
+        HomeRoute(text: ''),
```

Comment:
With ur logic if u need to pass text on many screens of ur app u need to create argument for it .